Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/PrefabFocusHandler.cpp

```diff
@@ -175,20 +175,20 @@ namespace AzToolsFramework::Prefab
         m_focusedInstance = focusedInstance;
         m_focusedTemplateId = focusedInstance->get().GetTemplateId();
 
-        AZ::EntityId containerEntityId;
-
-        if (focusedInstance->get().GetParentInstance() != AZStd::nullopt)
-        {
-            containerEntityId = focusedInstance->get().GetContainerEntityId();
-        }
-        else
-        {
-            containerEntityId = AZ::EntityId();
-        }
-            
         // Focus on the descendants of the container entity in the Editor, if the interface is initialized.
         if (m_focusModeInterface)
         {
+            AZ::EntityId containerEntityId;
+
+            if (focusedInstance->get().GetParentInstance() != AZStd::nullopt)
+            {
+                containerEntityId = focusedInstance->get().GetContainerEntityId();
+            }
+            else
+            {
+                containerEntityId = AZ::EntityId();
+            }
```

Comment:
nit: this could just be a ternary operator 🙂 

```
const AZ::EntityId containerEntityId = focusedInstance->get().GetParentInstance()
     ? focusedInstance->get().GetContainerEntityId();
     : AZ::EntityId();
```

The entity id can be const then too 😁

Reply:
Neat, fixed!